noir-projects/noir-protocol-circuits/crates/types/src/shared_mutable/scheduled_value_change.nr

```diff
@@ -9,10 +9,10 @@ mod test;
 // both the current value and the smallest block number at which the value might change - this is called the
 // 'block horizon'.
 pub struct ScheduledValueChange<T> {
-    pre: T,
-    post: T,
+    pub(crate) pre: T,
+    pub(crate) post: T,
     // Block at which `post` value is used instead of `pre`
-    block_of_change: u32,
+    pub(crate) block_of_change: u32,
```

Comment:
Had to do this change because now we need direct access to it when packing in `shared_mutable_values.nr`. Not great.